src/DependencyInjection/CmfSonataPhpcrAdminIntegrationExtension.php

```diff
@@ -61,6 +64,34 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader->load('main.xml');
         $loader->load('enhancer.xml');
+
+        $this->loadIvoryCKEditor($config['ivory_ckeditor'], $container);
+    }
+
+    protected function loadIvoryCKEditor(array $config, ContainerBuilder $container)
+    {
+        $container->setParameter('cmf_sonata_phpcr_admin_integration.ivory_ckeditor.config', []);
```

Comment:
i'd rather do 'if $this->isConfigEnabled... else setParameter []' in the main method